website/docs/fun/counting/overview.md

```diff
@@ -0,0 +1,82 @@
+## Features
+
+- Continuously count *forever*
+- Toggle whether users can count multiple times in a row; or just once, until another user counts
+- Assign a counting role to the current user
+- Toggle second chance: one-time save of the count
+- Customizable correct, warning, and incorrect emojis for reaction
+- Visual statistics including server, user, and leaderboard
+
+## Installation
+
+This package contains two custom commands: the [main command](https://github.com/H1nr1/yagpdb-cc/blob/master/src/fun/counting/main.go.tmpl), and [statistics](https://github.com/H1nr1/yagpdb-cc/blob/master/src/fun/counting/stats.go.tmpl).
+
+*The statistics command is not neccesary to add, but is a bonus.*
+
+## Main Command
+
+Add the [main command](https://github.com/H1nr1/yagpdb-cc/blob/master/src/fun/counting/main.go.tmpl) as a new custom command. The trigger is a regex trigger type with value `^(\d{1,}|\()`.
+
+::: caution
+
+Use channel restrictions to restrict this custom command to your counting channel.
+
+:::
+
+## Statistics
+
+Add [statistics](https://github.com/H1nr1/yagpdb-cc/blob/master/src/fun/counting/stats.go.tmpl) as a new custom command. The trigger is a command type with value `CStats`.
+
+:::info
+
+If you would like to customize the commands, see [here](overview/#configuration). For how to use it, please refer to the [usage](overview/#usage) section.
+
+:::
+
+### Configuration
+
+There are many configurable values in this package, to which only one needs to be configured: <sup>Main</sup> `$Admin`
+
+Optional configuration values include: <sup>Main</sup> `$CountTwice` `$RoleID` `$SecondChance` `$Leaderboard` `$CorrectEmoji` `$WarningEmoji` `$IncorrectEmoji` <sup>Stats</sup> `$LBLength`
+
+*How each of these values can be customly configured is described in the command itself*
```

Comment:
Please follow [our documentation guidelines](https://github.com/yagpdb-cc/yagpdb-cc/blob/master/WRITING-DOCUMENTATION.md) and adhere to the given template for this section.

Reply:
I have edited [Overview](https://github.com/H1nr1/yagpdb-cc/edit/master/website/docs/fun/counting/overview.md) to better suit the guidelines.
Should I show the separation between main-cc variable and stats variables?